internal/functions/efgs/cleanup-database.go

```diff
@@ -0,0 +1,24 @@
+package efgs
+
+import (
+	efgsdatabase "github.com/covid19cz/erouska-backend/internal/functions/efgs/database"
+	"github.com/covid19cz/erouska-backend/internal/logging"
+	"net/http"
+	"time"
+)
+
+//CleanupDatabase Remove old (more than 14 days) keys from database.
+func CleanupDatabase(w http.ResponseWriter, r *http.Request) {
+	var ctx = r.Context()
+	logger := logging.FromContext(ctx).Named("efgs.CleanupDatabase")
+	now := time.Now()
+	dateFrom := now.AddDate(0, 0, -14).Format("2006-01-02")
```

Comment:
I think it'd be worth it to have a constant `KeyValidityDays = 14` for usage both here and in upload (I don't remember a particular place but it's somewhere there, maybe in `database.go`).

Reply:
I couldn't find the correct const variable. So I had to create it.